jib-cli/build.gradle

```diff
@@ -0,0 +1,40 @@
+plugins {
+  id 'application'
+  id 'eclipse'
+}
+
+ext {
+  cliMainClass = 'com.google.cloud.tools.jib.cli.JibCli'
+}
+
+// use `run` to build and run the app
+// use `installDist` or `distZip` to create an installable application
+application {
+  applicationName = 'jib'
+  mainClassName = cliMainClass
+}
+
+dependencies {
+  implementation project(':jib-build-plan')
+  implementation project(':jib-core')
+  implementation project(':jib-core').configurations.implementation.dependencies
```

Comment:
We can also remove this line, you would only need this if you're trying to access classes in dependencies of jib-core. We do this in appengine-plugins-common kind of as a hack to not have to re specify the implementation dependencies (that is a little hacky, and maybe we should be more explicit about it -- but that's another issue).